UnityGLTF/Assets/UnityGLTF/Shaders/UnityStandardInput.cginc

```diff
@@ -1,3 +1,5 @@
+// Upgrade NOTE: replaced 'UNITY_INSTANCE_ID' with 'UNITY_VERTEX_INPUT_INSTANCE_ID'
```

Comment:
Personally I don't think .cginc file should be included in the PR for compatibility concerns.

Reply:
How to modify the PR?

Reply:
https://stackoverflow.com/questions/9790448/how-to-update-a-pull-request-from-forked-repo